CMakeLists.txt

```diff
@@ -437,6 +437,7 @@ add_subdirectory(src)
 if(CESIUM_OMNI_ENABLE_TESTS)
     enable_testing()
     add_subdirectory(tests)
+    add_subdirectory(cesiumTestsExtension)
```

Comment:
It's ok that the folder is called `cesiumTestsExtension` in this PR, but once we migrate the existing tests we should rename the folder back to `tests`.